### ISSUE TITLE
Repeated keywords in the autogenerated documentation

### ISSUE THREAD

**Issue by stigrj** [dev-cafe/parselglossy#49]
* parselglossy version:  b28084a
* Python version: 3.7.0
* Operating System: Ubuntu-18.10

### What I Did
The following `template.yml`
```
keywords:
  - name: foo
    type: int
    docstring: foo
  - name: bar
    type: int
    docstring: bar
  - name: baz
    type: int
    docstring: baz
```
is documented using the CLI command `parselglossy doc template.yml` to give:
```
**Keywords**
 :foo: foo

  **Type** ``int``

**Keywords**
 :foo: foo

  **Type** ``int``

 :bar: bar

  **Type** ``int``

**Keywords**
 :foo: foo

  **Type** ``int``

 :bar: bar

  **Type** ``int``

 :baz: baz

  **Type** ``int``

```

**Comment by robertodr** [comment]
Is this only for top-level keywords? Or also for keywords in sections?

**Comment by stigrj** [comment]
Same for sections

**Comment by robertodr** [comment]
OK, thanks for the report. Looking at it now.

**Comment by robertodr** [comment]
Fixed with #52
